pkg/usagetracker/tracker.go

```diff
@@ -38,25 +46,49 @@ func (c *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 
 	c.InstanceRing.RegisterFlags(f, logger)
 	c.PartitionRing.RegisterFlags(f)
+	c.EventsStorage.RegisterFlags(f)
 	c.SnapshotsStorage.RegisterFlagsWithPrefixAndDefaultDirectory("usage-tracker.snapshot-storage.", "usagetrackersnapshots", f)
 
 	f.DurationVar(&c.IdleTimeout, "usage-tracker.idle-timeout", 20*time.Minute, "The time after which series are considered idle and not active anymore. Must be greater than 0 and less than 1 hour.")
 }
 
+func (c *Config) Validate() error {
```

Comment:
We can also validate the IdleTimeout < 1h here now (instead of the constructor)

Reply:
Done in https://github.com/grafana/mimir/pull/10137/commits/80b18184698a4112644e7d459899dc6db9eaca64